hepdata/modules/records/templates/hepdata_records/formats/datacite/datacite_container_submission.xml

```diff
@@ -35,7 +35,7 @@
     <resourceType resourceTypeGeneral="Collection">Collection</resourceType>
     <version>{{ overall_submission.version }}</version>
     <descriptions>
-        <description descriptionType="Abstract">{{ overall_submission.data_abstract | truncate(5000) }}</description>
+        <description descriptionType="Abstract">{% if overall_submission.data_abstract %}{{ overall_submission.data_abstract | truncate(5000) }}{% else %}{{ overall_submission.data_abstract }}{% endif %}</description>
```

Comment:
Do we need the else in here (and in similar lines below)? If the abstract is empty then there's nothing to display?

Reply:
The `else` condition writes out `None` which I thought was preferable to being completely empty.

Reply:
Now changed to "No data abstract available." etc. instead of `None`.